Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -155,9 +147,66 @@ struct FrameworkBundleAssembler {
     }
 
     private func copyResources() throws {
-        if let resourceBundlePath = frameworkComponents.resourceBundlePath {
-            let destinationPath = frameworkBundlePath.appending(component: resourceBundlePath.basename)
-            try fileSystem.copy(from: resourceBundlePath, to: destinationPath)
+        func copyInfoPlist() throws {
+            let sourcePath = frameworkComponents.infoPlistPath
+            let destinationPath = frameworkBundlePath.appending(component: "Info.plist")
+            try fileSystem.copy(from: sourcePath, to: destinationPath)
+        }
+
+        /// Retruns the resulting, copied resource bundle path.
+        func copyResourceBundle() throws -> TSCAbsolutePath? {
+            if let sourcePath = frameworkComponents.resourceBundlePath {
+                let destinationPath = frameworkBundlePath.appending(component: sourcePath.basename)
+                try fileSystem.copy(from: sourcePath, to: destinationPath)
+                return destinationPath
+            } else {
+                return nil
+            }
+        }
+
+        // Move PrivacyInfo.xcprivacy to expected location (if exists in the resource bundle)
+        //
+        // ref: https://developer.apple.com/documentation/bundleresources/adding-a-privacy-manifest-to-your-app-or-third-party-sdk#Add-a-privacy-manifest-to-your-framework
+        func movePrivacyInfoIfNeeded(
```

Comment:
I feel`move` has no enough description because we cannot expect when it should be called and get well about how `resourceBundlePath` and `relativePrivacyInfoPath` for destination. How about naming like below?


```suggestion
        func movePrivacyInfoToRequiredLocationForThirdPartyFrameworkIfNeeded(
```

... very long lol

Reply:
How about something like this? `func extractPrivacyInfoFromResourceBundleToFrameworkIfExists`

Reply:
Renamed at b7ab22f4782b8b05cb410c1e0603e5d3da0c817b